nms/app/packages/magmalte/app/views/network/NetworkEpc.js

```diff
@@ -145,7 +145,6 @@ export function NetworkEpcEdit(props: EditProps) {
           <AltFormField label={'IP Allocation Mode'}>
             <Select
               variant={'outlined'}
-              fullWidth={true}
```

Comment:
what does this change do ?

Reply:
The width of this select input was not matching other inputs width.
![Screen Shot 2020-10-06 at 5 38 42 AM](https://user-images.githubusercontent.com/26038920/95433387-25bb5600-0950-11eb-8551-5114f4701e94.png)